4.8/Dockerfile

```diff
@@ -6,17 +6,15 @@ RUN groupadd --gid 1000 node \
 # gpg keys listed at https://github.com/nodejs/node
 RUN set -ex \
   && for key in \
-    9554F04D7259F04124DE6B476D5A82AC7E37093B \
     94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
-    0034A06D9D9B0064CE8ADF6BF1747F4AD2306D93 \
     FD3A5288F042B6850C66B31F09FE44734EB7990E \
     71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
     DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
-    B9AE9905FFD7803F25714661B63B535A4C206CA9 \
     C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+    B9AE9905FFD7803F25714661B63B535A4C206CA9 \
```

Comment:
Let me rephrase that, now that I've thought about it: note the older keys is good, but at this point the only key that should be there is `9554F04D7259F04124DE6B476D5A82AC7E37093B` since the others are for Node.js releases that are long gone and were never use for v4 and up